jacodb-analysis/src/main/kotlin/org/jacodb/analysis/codegen/language/impl/JavaLanguage.kt

```diff
@@ -410,7 +422,7 @@ class JavaLanguage : TargetLanguage {
                 }
             }
             appendLocalsAndSites(constructor)
-        }
+        }, true)
```

Comment:
constants athat are passed as parameter should be prefixed with its parameter name - `parameterName = true`